src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -46,6 +46,13 @@ public LogicManager(Model model, Storage storage) {
     public CommandResult execute(String commandText) throws CommandException, ParseException {
         logger.info("----------------[USER COMMAND][" + commandText + "]");
 
+        ReadOnlyZooKeepBook currentUpdate = new ZooKeepBook();
+        boolean isEmpty = true;
```

Comment:
What is this isEmpty for? Looks a little out of place.

Reply:
This confirms if the redo stack is empty since there is a check below from lines 65-67.

Reply:
Can we just give the redo stack a getSize() or isEmpty()? Otherwise this boolean is like dangling out of nowhere.

Reply:
Ok removed